### DISCUSSION TITLE
Problems with AspectOS

### DISCUSSION THREAD

**Discussion by AIGamesDeveloper** [AspectOSTeam/AspectOS#15] [AT Announcements]
Hello everybody!

I am here to announce some things and they aren't good announcements. Over the past couple of days, AspectOS has seen a decline in activity and we are getting nothing done again. The thread hasn't had activity in 24 hours until the most recent post stating that I closed the thread temporarily. But will this be temporary? I don't know. With the decline in activity, it may as well be for a while, most likely until the end of summer. I made this so you know why I closed the thread. I would never close the thread out of context.

## Team Development

Now I love AspectOS and I always wanted for it to grow and for it to be its best. But how will I be able to do that if I have a team that has other things to do? I understand that they aren't robots and they have other things to do, but when it gets to the point where we have everybody on break for the summer when we most need people, I'd rather go solo and work on the operating system myself. And I'm serious. I understand what I am saying and I don't have to break down ideas for AspectOS because I already know. I don't have to manage so many things and I don't have to wait for tasks to get finished when I can finish them. 

Now, there might be some misunderstandings. And I am here to just say some things before you start asking questions or accusing me of anything. Yes, I still care about AspectOS and the community, but the community is dead and I think that I need some time to work on this OS alone and not wait for other people. No, I did not use anyone for my own gain. I would never run off like that and I will continue to work with my team on AspectOS in the future. But right now is not a good time to work on it. And no, AspectOS is not canceled. 2.0 is still in development and it will still have the same release date. I am not delaying the date and I will work on the OS myself to keep that release date.

## External Content

At this point, I don't know what to do with external content. Things like the website will stay but some other things like the Wiki will have to be reconsidered. I have been a fool to allow all of these things and I have to think about the effects of these causes that I have committed. Contributors will still be able to work on external websites. I cannot do anything to stop them. But I doubt that anyone will be able to work on those considering that everybody has left for the summer. Who doesn't love the summer? I can't stop something so fun from happening, because I would become an overprotective parent.

## The Future of AspectOS

As of right now, the future doesn't start here in the present because of how dead AspectOS has become. It will take months for the team to get back together and honestly, I would wanna create a reboot of the team and remove everybody except the characters who were in the past show and will be in the new show. Basically, everybody might get removed except for some people. This will cause controversy and lots of people will be mad at me but this is the future. It might be good for some people and it might sound messed up for others. This is what the decline in activity has come to.

Now if I go with this plan, then everybody except the following will be removed

- GAMS2
- Dream-123-123
- anthrofurever
- kkidslogin
- yooit_32
- Tetra22
- Lily-Lavender

It's not like I hate everybody, but that you lack the skill needed to stay in AspectOS, like effort or activity. And no, this doesn't violate the rules because it is not unfair and this is a logical reason. And since I am AIGamesDeveloper, I say that this is a logical reason.

> We have the right to remove you from the collaboration for a logical reason. If you think you have been removed unfairly by the BoD, let @AIGamesDeveloper know and he will decide if your removal was for a logical reason.

Everybody who helped will still be credited on the website and the website will be linked to the project to save space for the description and the project itself. But you will be removed from the collaboration until applications open again, which won't be for a couple of months instead, we will be sending invites to people for them to join.

## Changes Coming

Because of the actions that I am proposing, some changes will occur. First, the BoD will be removed due to the lack of employees. Only 3 managers will occur and those are AIGamesDeveloper, GAMS2, and Dream-123-123 if they decide to stay. Next, most tasks will be removed and some will be added to make us more comfortable

Instead of having applications open to the public, we will be sending invites to those who are eligible for AspectOS. It might be a risky move, but it will help find better contributors for AspectOS and therefore, we will have a better working team since the person matches our standards. Once in a while, we will have applications open to the public, but they will only be open for 48 hours max.

The rules will be rewritten to match our standards, but they won't drastically be changed. It's more of a revision than a rewrite. Things such as the post count, the number of off-topic posts, and what we require for art, code, and more, will change to stricter standards, but not extremely strict. We still want contributors to have fun.

And the biggest one of all that might confuse some people is to reread the sentence to make sure they haven't gone crazy. Make sure to sip a glass of water and read the next sentence because it will surprise you. AspectOS may be moving away from Aspect Inc. This might sound like a dumb idea, but this could be an opportunity for Aspect Inc. to grow and we can find other ways to succeed other than the one thing that is making it live which is this operating system. So if you are a "company" and you want to have AspectOS as part of your company, then you can ask me and we can try to negotiate.

## Conclusion

Yes, this is short and I know that I could've gone more into depth about why I closed the thread, but that is about as deep as I can get right now. I just think that we all need a break, and some may have longer breaks than others. And because of this, AspectOS needs to rest as well. I am sorry for those who still wanted to work on AspectOS. I will try my best to make sure that I open this thread again and that I have people working again. 

When will the thread open again? I don't know right now. It may be closed for a couple of weeks or at the most, two months. Contributors won't be able to share anything with the team and I will work on the OS by myself so that when we come back, I have made some progress and we can continue from there to make one of the best operating systems we have created.

I am sorry that this happened two days before Aspect's First Birthday. I will still celebrate the 1st birthday and contributors will also celebrate it as well, just not on the thread. Most likely on the comments of the birthday project. It may be something that is disappointing to some people but others are excited about it (you know who you are)

I can't wait to see this thread return to its golden days but as a reboot. The future does not start here. But it will soon.

**Comment by qwertyxbutnot** [comment]
Hahahahaaaaaaa WHY. 

When the only reason you're on Scratch at this point temporarily shuts down and summer break is probably your last chance to do anything on Scratch for a while:

Did he died?
Yes. He is died.
(Btw this is my alt)

**Reply by ghost** [reply]
This is the same for me. I found something else though to keep me on scratch. You better find something too.

**Reply by AIGamesDeveloper** [reply]
I am sorry to hear that. I have confirmed the date of the thread reopening and it will be on July 1, 2022. It may be earlier and it may be when all breaks are over.

**Reply by ufgtftcbtcut** [reply]
July or earlier? Now thats great!

**Reply by AIGamesDeveloper** [reply]
> July or earlier? Now thats great!

I can't tell if this is sarcastic or not

If not sarcastic: Yes! It's a good thing that this isn't going to be closed until school starts! I do think that having no school would mean increased activity for AspectOS

If sarcastic: I understand that many people want to join back but there are so many people on break and I just have decided to put everybody on break

> July or earlier? Now thats great!

**Reply by TheMowgliMan** [reply]
The only reasons I'm on Scratch (in chronological order) is A) AspectOS Stuff B) Conlanger Studio C) Country Clash WRP

**Reply by ufgtftcbtcut** [reply]
> > July or earlier? Now thats great!
> 
> I can't tell if this is sarcastic or not
> 
> If not sarcastic: Yes! It's a good thing that this isn't going to be closed until school starts! I do think that having no school would mean increased activity for AspectOS
> 
> If sarcastic: I understand that many people want to join back but there are so many people on break and I just have decided to put everybody on break
> 
> > July or earlier? Now thats great!
> 
> 

Not sarcastic lol

**Comment by ghost** [comment]
I think i's logical to only have people who can commit the best they can in an organization.

**Reply by AIGamesDeveloper** [reply]
I agree. I want the best of the best and I have always said this but never actually did that. I am serious this time and I will only invite the best of the best

**Reply by ghost** [reply]
Yeah. Also, whenever I was part of Aspect, I noticed that the activity checks were a pretty bad system cause they were pretty rare.

**Reply by AIGamesDeveloper** [reply]
yes, and they weren't the bset thing

**Comment by ufgtftcbtcut** [comment]
Will the removed people be able to use a form to rejoin?
And the reason so many people became inactive in the first place is because you hired too many people, so you dont have time to manage all the tasks for everyone and some start to get treated less or ignored, like comments being ignored for months and the todolist not updating so some work is put into the trash, like someone makes a really good app but its not said to be done in the todo and a new member comes in and makes a worse app but that one gets accepted and added, frustrating the original ones owner (who would probably go on to find an os with better management and quits aspect) and decreasing the overall quality if the os, causing them to be inactive. some inactive people goes do something else that someone else in the team may be interested in and this causes a chain of people becoming inactive while still being a member of aspect and forgetting about it
I suggest you keep it on as a collab, dont make it a solo, remake the team but only hire the veteran people who have worked with aspect in the first, like, 3 months or really experienced people
This is not a rant or hate message, i just feel like you should change how aspect collaboration works. And im an old aspect member who has made like 5 apps for aspect in its first 3 months but none of them are even acknowledged, so i quitted. Ive now moved on to C++, C, and JS, and ive publicly told that. Guess who i am

**Reply by AIGamesDeveloper** [reply]
Most of them will be reinvited. We just need the base contributors back and then we will slowly invite everybody back. Think of it as a huge activity check for the OS.

And yes I agree, with so many things happening at once and me accepting people to join the OS, it has become a struggle in my life, and in my 12th year of living, it was so hard and there was a lot of pressure put on me. I am sorry if you were ignored by me or any of the BoD since they also have the same power as my power. 

And let me guess, RobloxianMinecraft?

**Reply by ufgtftcbtcut** [reply]
How did you know? Anyways, yes, im RM. Ill be willing to rejoin aspect if you invite me
And sorry if I sounded rude there

**Reply by AIGamesDeveloper** [reply]
> How did you know? Anyways, yes, im RM. Ill be willing to rejoin aspect if you invite me And sorry if I sounded rude there

Your repositories
And you might be one of the 15 people who I will invite

**Comment by alphacodes26** [comment]
Can I please get AspectOS contributor invitation whenever needed?
(Im abcde, ive been waiting for months to get back)

**Reply by AIGamesDeveloper** [reply]
Yes, but you will have to go to Aspect School so people know how we do things before we actually get started with the development of the OS. It may be a side thing but it will be beneficial in the long run

**Comment by TheMowgliMan** [comment]
Hmmm.... Alright. I think separating Aspect Inc. and AspectOS is a risky move however. Thanks though for having me up there on the automatically-back list!

**Reply by AIGamesDeveloper** [reply]
This is all experimental, to be honest. If moving to a different "company" doesn't work, I can always remove the OS from the "company" and move back into AspectOS

I do think that Aspect Inc. should have another successful thing other than AspectOS. I feel like Aspect Inc. is only feeding off of AspectOS' sucess

**Reply by TheMowgliMan** [reply]
I feel like the reason that Aspect Inc. is not major is that none of it's projects are as exciting as AspectOS. I cannot remember a single one off the top of my head. They just aren't major or memorable, so they wither and die. It's AINC's own fault, and AspectOS has no blame.

By the way, I suggest adding another good coder who isn't going to be BOD to the list. Currently, yooit_32 is the only one there on your auto-accept list, besides me maybe but I hardly ever code in Scratch anymore. While I do not know most of the Aspecters very well, I suggest adding in Tigitaal (Tycho) or abcde back. There may be better options, but I cannot think of any. Yes, I know that neither of them are in, and that they've had their share of mischief. If you don't want that, and don't mind a less active coder, ToastersUnited (NILL2021) is a good choice.

**Reply by TheMowgliMan** [reply]
In addition, you're a better-than-average leader. It's very hard to keep a collab active for an extended period like AspectOS was. So when a AINC product is handed off to someone else, it dies after a while. I suggest possibly killing AINC entirely and focusing on the OS.

**Reply by AIGamesDeveloper** [reply]
> I feel like the reason that Aspect Inc. is not major is that none of its projects are as exciting as AspectOS. I cannot remember a single one off the top of my head. They just aren't major or memorable, so they wither and die. It's AINC's own fault, and AspectOS has no blame.
> 
> By the way, I suggest adding another good coder who isn't going to be BOD to the list. Currently, yooit_32 is the only one there on your auto-accept list, besides me maybe but I hardly ever code in Scratch anymore. While I do not know most of the Aspecters very well, I suggest adding in Tigitaal (Tycho) or abcde back. There may be better options, but I cannot think of any. Yes, I know that neither of them is in and that they've had their share of mischief. If you don't want that and don't mind a less active coder, ToastersUnited (NILL2021) is a good choice.

ConvergeOS was the 2nd biggest thing of Aspect Inc, and that is dead and was a backup to AspectOS, so you are right, there isn't anything going on with Aspect Inc at the moment and aOS shouldn't take the blame, but that's the point. The only thing we have right now is aOS that is still alive. We need more **active** threads for aINC that won't die within two weeks. We need projects that will stay alive for a while.

Also, I might remove the BoD. I don't think having the BoD is the best thing for this OS and I feel like the BoD has caused some problems (not the people, but the idea itself) and I don't want any more problems to occur with it. But I will consider the three that you chose. I also like those coders and how they do things for aOS.

**Reply by AIGamesDeveloper** [reply]
> In addition, you're a better-than-average leader. It's very hard to keep a collab active for an extended period like AspectOS was. So when a AINC product is handed off to someone else, it dies after a while. I suggest possibly killing AINC entirely and focusing on the OS.

I agree. I haven't seen a collaboration last this long besides Meower. But that also stopped because it broke the rules, so as of right now, we are the longest living collaboration that hasn't been closed yet and is still active on Scratch as far as I know.

I do think that handing the OS to another company may be a bad thing because of the actions that the company can cause, but killing aINC now? I don't know about that... I have mixed feelings about that and if I were to shut aINC down, then those collaborations would crumble because they would lose the benefits of aINC. 

I could remove the partner thing since they are requesting many things from us and honestly, I don't like it.

**Comment by TheMowgliMan** [comment]
Yeah, partners are hard.

**Reply by AIGamesDeveloper** [reply]
Exactly, and it was a bad idea in the first place. Not only will I remove it from aINC, I will remove it from aOS

**Comment by TheMowgliMan** [comment]
Removing the bod isn't a bad idea, besides keeping you. I have a new
project with no leadership positions.

**Reply by AIGamesDeveloper** [reply]
I will only have 2 other managers, GAMS and Dream.

**Comment by TheMowgliMan** [comment]
I remember when you added ainc. It didn't have much point except as a
storage bin. I think you should cut your losses and remove it.

**Reply by AIGamesDeveloper** [reply]
Aurora Inc. will not get removed and I will try to compose a plan to bring it back.

**Comment by ufgtftcbtcut** [comment]
I think you should have aspect remain in ainc but rename aspect inc to something else thats not aspect inc but is related to "aspect", maybe points inc.
So it doesnt sound like the only product is aspectos but it still shows that aspect is the main product

Also can we merge urmoji and cornmoji together? If yes then thats 4 emoji galleries into 1 (because design center/flags and one emojis added to cornmoji)

**Reply by AIGamesDeveloper** [reply]
I was already thinking of renaming aINC to Aurora Inc.

and right now, Urmoji is being reconsidered.

**Reply by ufgtftcbtcut** [reply]
if we really merge, how about packmoji?
the names not taken and its a bit catchy

imagine like
Packmoji - The coolest emojis - By Aurora Inc. and Pizn

**Comment by AIGamesDeveloper** [comment]
Announcement: AspectOS will open on June 15, 2022!
I have chosen 12 people to stay in AspectOS. These are the best of the best and they are active and will work on the OS. If you are not on here, then you have not shown that you are worthy to stay in AspectOS. The rest of the 13 people will be decided. I have made a deal with Citrus Labs (The people who work on OrangeOS) and the creator said that they will invite some oOS contributors to aOS. I don't know if this will be the best thing but we are willing to try new things to see if they will work in the future.

1. GAMS2
2. Dream-123-123
3. anthrofurever
4. kkidslogin
5. yooit_32
6. Tetra22
7. Lily-Lavender
8. ToastersUnited
9. RobloxianMinecraft
10. abcde26
11. Tigitaal
12. Sil3nci0

**Comment by GAMS2** [comment]
I would like to STAY

**Reply by AIGamesDeveloper** [reply]
don't worry, you are already in the invite list

**Comment by qloakongithub** [comment]
can i stay

also, I've already given a reason for this thread starting to fail. people don't really have much to do. I've given multiple improvements that you've just overlooked.

**Reply by AIGamesDeveloper** [reply]
Sorry, I have already chosen people who are active and are willing to work on AspectOS. I try my best to not overload the contributor list and to make sure that we have enough tasks for everybody as I come up with new ideas. And to be honest, you didn't even help at all for AspectOS in the past months.

If a manager wants to invite you to AspectOS, they can since we are starting an invite-only system.

And besides, you were already thinking of leaving AspectOS, so why ask to rejoin now?

And you sent me death threats and cursed at me, so why the heck should I let you in now? You have been permanently banned.

**Comment by AIGamesDeveloper** [comment]
Alright everybody, thank you for your opinions and your feedback. I will close this now. If you have any more questions, feel free to make a new discussion [here.](https://github.com/AspectOS/AspectOS/discussions/new) Thanks!
